src/neo/Ledger/TransactionVerificationContext.cs

```diff
@@ -13,8 +13,16 @@ public class TransactionVerificationContext
         /// </summary>
         private readonly Dictionary<UInt160, BigInteger> senderFee = new Dictionary<UInt160, BigInteger>();
 
+        /// <summary>
+        /// Store oracle responses
+        /// </summary>
+        private readonly Dictionary<ulong, UInt256> oracleResponses = new Dictionary<ulong, UInt256>();
+
         public void AddTransaction(Transaction tx)
```

Comment:
Why don't remove `CheckTransaction` and Change Add to `bool TryAdd` ?

Reply:
Because `CheckTransaction` is called in `Transaction.Verify()`.